frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/DetailsCard.tsx

```diff
@@ -17,9 +17,9 @@ import { BareMetalHostKind } from '../../../types';
 import { BareMetalHostDashboardContext } from './BareMetalHostDashboardContext';
 
 const DetailsCard: React.FC<DetailsCardProps> = () => {
-  const { obj, machine, nodes } = React.useContext(BareMetalHostDashboardContext);
+  const { machine, nodes } = React.useContext(BareMetalHostDashboardContext);
   const node = getMachineNode(machine, nodes);
-  const hostName = getName(obj);
+  const hostName = getName(node);
```

Comment:
This should be kept intact as `hostName` is used in the first `DetailItem` below.